distributions/demo-server/src/config/client/nginx.conf

```diff
@@ -0,0 +1,65 @@
+user  nginx;
```

Comment:
License header and maybe a quick line of documentation

Reply:
Sure will do

Reply:
License header still doesn't appear for me.

Reply:
Sorry added now